pipelines/nextflow/modules/download/download_genbank.nf

```diff
@@ -13,21 +13,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-nextflow.enable.dsl=2
 
 process DOWNLOAD_GENBANK {
-    label "Sequence_genbank_file"
-    tag "${meta.accession}"
+    tag "${meta.production_name}"
     label 'default'
+    storeDir "$cache_dir/genbank/${meta.accession}"
+    afterScript "sleep $params.storeDir_latency"
 
     input:
         val(meta)
+        val(cache_dir)
```

Comment:
not a path?

Reply:
Not in this case: this cache_dir is a value used by Nextflow itself but not by the process. The "path" is for the files that we need to to access/manipulate within the process (so that the files can be "staged").